op-program/client/interop/interop.go

```diff
@@ -155,6 +156,10 @@ func deriveOptimisticBlock(logger log.Logger, bootInfo *boot.BootInfoInterop, l1
 	if err != nil {
 		return types.OptimisticBlock{}, fmt.Errorf("no chain config available for chain ID %v: %w", chainAgreedPrestate.ChainID, err)
 	}
+	l1ChainConfig, err := bootInfo.Configs.L1ChainConfig(eth.ChainIDFromBig(rollupCfg.L1ChainID))
+	if err != nil {
+		return types.OptimisticBlock{}, fmt.Errorf("no chain config available for chain ID %v: %w", eth.ChainIDFromBig(rollupCfg.L1ChainID), err)
```

Comment:
(nit) Might be helpful to specify the error is for an L1 chain:
```suggestion
		return types.OptimisticBlock{}, fmt.Errorf("no chain config available for L1 chain ID %v: %w", eth.ChainIDFromBig(rollupCfg.L1ChainID), err)
```

Reply:
0aa46fd4878b5b18c1353e5abcf0a6d36a907c48 and 47df662db0245d62b170075fb2bb3609c6ea54ad